lua/lua-core/intro-to-lua/what-to-expect.md

```diff
@@ -11,10 +11,11 @@ type: normal
 
 Congratulations on practicing with your first few lines of Lua! Sure, it may have been a simple hello statement, but **you've made progress!** 🎉
 
-This may seem underwhelming at first, but Lua can lead to very complex ideas and logic! 
+As you continue in your journey of learning Lua, you'll grasp a strong understanding and be able to flexibly apply the principles of Lua due to its ease of integration.

```

Comment:
I'm not sure this was an improvement, seems like it says the same thing as the sentence on the left but in more words :) 

What if we just remove it? 

This paragraph seems to jive well with the one below talking about a roblox rotation object

Reply:
I agree that it was saying the same thing, but I wanted to emphasize again on why learning Lua could be beneficial, as some comments were asking.

So in the change below, as your recommendation, I removed the first paragraph, but included some extra words describing the applicability of Lua to make it concise:

"This may seem underwhelming at first, but Lua can lead to very complex ideas and logic **applicable in many areas of coding**! For instance, here's a small rotation object taught in the Roblox course!"